pandas/tests/reshape/test_pivot.py

```diff
@@ -2598,3 +2598,11 @@ def test_pivot_not_changing_index_name(self):
         expected = df.copy(deep=True)
         df.pivot(index="one", columns="two", values="three")
         tm.assert_frame_equal(df, expected)
+
+    def test_pivot_table_empty_dataframe_correct_index(self):
+        # GH 21932
+        df = DataFrame([], columns=["a", "b", "value"])
+        pivot = df.pivot_table(index="a", columns="b", values="value", aggfunc="count")
+
+        expected = pd.Index([], dtype="object", name="b")
+        tm.assert_index_equal(pivot.columns, expected)
```

Comment:
I'd suggest you run `pre-commit`, please check the contributing guide for how to do that

Reply:
thx for the head up, done :)